setup.py

```diff
@@ -335,10 +383,8 @@ def do_setup():
             'all_dbs': all_dbs,
             'atlas': atlas,
             'async': async_packages,
-            'azure_blob_storage': azure_blob_storage,
-            'azure_data_lake': azure_data_lake,
-            'azure_cosmos': azure_cosmos,
-            'azure_container_instances': azure_container_instances,
+            "aws": aws,
```

Comment:
```suggestion
            'aws': aws,
```

Matching quote style